inc/class-client.php

```diff
@@ -28,8 +28,8 @@ class Client {
 	 */
 	public function __construct() {
 
-		$this->content_filter_locations();
-		$this->link_filter_locations();
+		add_action('init', array( $this, 'content_filter_locations'));
```

Comment:
@richdacuban nitpick, but can you even out the spacing in these two lines?

- 1 space between `(` and `'init'`
- 1 space between `_locations'` and `)`
- 1 space between `)` and `)`

Reply:
@douglas-johnson Apologies, that was messy of me. Edited a/p directions.